include/deal.II/grid/cell_id_translator.h

```diff
@@ -187,16 +211,27 @@ namespace internal
                     dim == Accessor::structure_dimension,
                   "The information can only be queried for cells.");
 
-    types::global_cell_index id = 0;
+    Assert(!tree_sizes.empty(), ExcNotImplemented());
```

Comment:
I added some asserts, in the case that someone calls these function with the new default constructor.